### ISSUE TITLE
Add visual indicator when a process is running?

### ISSUE THREAD

**Issue by TokugawaTFC** [Gwyrdain/apprentices-workshop#96]
I clicked the 'Review' button and it looked like nothing was happening. Can we get a visual indicator of some sort when it takes awhile to do something?

This also happened when I was trying to download the file.

**Comment by Gwyrdain** [comment]
Agreed (thought I don't know how yet).  Part of the slowness is probably just the degree of horsepower sitting behind this free hosting.
